c/macOS/src/main.c

```diff
@@ -0,0 +1,90 @@
+//
+//  main.c
+//  mptcp-network-client
+//
+//  Created by Anthony Doeraene on 08/08/2024.
+//
+
+#include <stdio.h>
+#include <string.h>
+#include <Network/Network.h>
+
+dispatch_semaphore_t semaphore;
+
+void get(char *path, nw_connection_t conn, dispatch_queue_t queue){
+    char req[1024];
+    int size = sprintf(req, "GET %s HTTP/1.0\r\nUser-Agent: curl/1.0.0\r\n\n", path);
```

Comment:
(detail: can you not use `req = "..."; size_t size = sizeof(req) - 1;`)

Reply:
I may be missing some piece of information, but how can you do string interpolation in C without using sprintf ?

Reply:
My bad, I didn't see the `path` at the end. Too many `\` in this string :)
All good then!